From fa5fbbf788b804dab4d135bfed6e8d779c2054e6 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Fri, 28 Sep 2018 20:31:38 +0800
Subject: [PATCH 4748/5242] MLK-19687 ARM64: dts: fsl-imx8mm-evk: add
 off-on-delay for usdhc2

commit  c72818eb0e9cb463ebfe85bd63b34b0fce5f8c78 from
https://source.codeaurora.org/external/imx/linux-imx.git

Make sure the time between power off and power on sd card meet the
sd SPEC, otherwise after suspend and resume, SD3.0 card will no longer
be recognized as SD3.0, but SD2.0

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index ce88171e..af0f759 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -37,6 +37,7 @@
 			regulator-min-microvolt = <3300000>;
 			regulator-max-microvolt = <3300000>;
 			gpio = <&gpio2 19 GPIO_ACTIVE_HIGH>;
+			off-on-delay = <20000>;
 			enable-active-high;
 		};
 
-- 
1.7.9.5

